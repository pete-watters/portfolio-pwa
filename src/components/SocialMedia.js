import React, { memo } from 'react';
import Icon from './Icon';
import CONFIG from '../constants';

const SocialMedia = () =>
    <>
      <Icon icon="github" href={CONFIG.SOCIAL.GITHUB} />
      <Icon icon="linkedIn" href={CONFIG.SOCIAL.LINKEDIN} />
      <Icon icon="stackOverflow" href={CONFIG.SOCIAL.STACKOVERFLOW} />
      <Icon icon="medium" href={CONFIG.SOCIAL.MEDIUM} />
      <Icon icon="instagram" href={CONFIG.SOCIAL.INSTAGRAM} />
      <Icon icon="twitter" href={CONFIG.SOCIAL.TWITTER} />
    </>;

export default memo(SocialMedia);