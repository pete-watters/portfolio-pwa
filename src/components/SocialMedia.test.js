import React from 'react';
import { shallow } from 'enzyme';
import SocialMedia from './SocialMedia';

describe('SocialMedia', () => {
  it('should render the component correctly', () => {
    expect(shallow(<SocialMedia />)).toMatchSnapshot();
  });
});
