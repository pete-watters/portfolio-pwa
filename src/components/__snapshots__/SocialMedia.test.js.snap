// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialMedia should render the component correctly 1`] = `
<Fragment>
  <Memo(Icon)
    href="https://github.com/pete-watters"
    icon="github"
  />
  <Memo(Icon)
    href="https://www.linkedin.com/in/pete-watters/"
    icon="linkedIn"
  />
  <Memo(Icon)
    href="https://stackoverflow.com/users/1365580/peadar"
    icon="stackOverflow"
  />
  <Memo(Icon)
    href="https://medium.com/@pete_watters"
    icon="medium"
  />
  <Memo(Icon)
    href="https://www.instagram.com/watterspete/"
    icon="instagram"
  />
  <Memo(Icon)
    href="https://twitter.com/watters_pete"
    icon="twitter"
  />
</Fragment>
`;
